File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)

{

// Decleration for height so height can be used in loop    
int height = 0;  
    
// Asks for the height and asks again when number isn't between 1 and 8
do 
    { 
       height = get_int("Height: "); 
    }

while (height < 1 || height > 8);
    
    {
        // nested for loops to generate pyramids when height is between 1 and 8
        for (int row = 1 ; row <= height ; row++)
        {
        
        for(int i = height - row -1 ; i >= 0; i--)
        {
            printf(" ");
        }
        
        for(int i = 0 ; i < row; i++)
        {
            printf("#");
        }
            
        printf("  ");
             
        for(int i = 0 ; i < row ; i++)
        {
            printf("#");
        }
                     
        printf(" \n");
            
        }
    }
}

File: mario.py
from cs50 import get_int


def main():
    height = get_positive_int("Positive integer: ")

    # Loop over lenght of tower
    for row in range(height):

        space = height - row - 1
        for i in range(space, 0, -1):
            print(' ', end='')

        for j in range(row + 1):
            print('#', end='')

        print('  ', end='')

        for k in range(row + 1):
            print('#', end='')

        print('')

# Define new function to prompt for positive integer between 1 and 9
def get_positive_int(prompt):
    while True:
        n = get_int(prompt)
        if n > 0 and n < 9:
            break
    return n


if __name__ == "__main__":
    main()